app/uk/gov/hmrc/transitmovements/models/mongo/read/MongoMovementSummary.scala

```diff
@@ -58,7 +60,8 @@ case class MongoMovementSummary(
       movementReferenceNumber,
       localReferenceNumber,
       created,
-      updated
+      updated,
+      isTransitional.getOrElse(true)
```

Comment:
I don't think so it is required? As this is for get call from common-transit-convention-traders?